src/Config/Utilities/FileUtilities.cs

```diff
@@ -42,7 +42,7 @@ public static byte[] ComputeHash(IFileSystem fileSystem, string filePath)
         int runCount = 1;
 
         // Maximum 2^_runLimit seconds of wait time due to retries.
```

Comment:
Change the variable name in the comment
```suggestion
        // Maximum 2^RunLimit seconds of wait time due to retries.
```